agent/credentials/providers/credentials_filename_windows.go

```diff
@@ -0,0 +1,24 @@
+//go:build windows
```

Comment:
You dont need this

Reply:
I agree but I followed the convention followed across in this repo.

Reply:
IMO we should not add tags that golang handles by default. Even if its done in other places in the repo